src/terminatesteadystate.jl

```diff
@@ -21,14 +21,20 @@ function allDerivPass(integrator, abstol, reltol)
     else
         any((abs.(testval) .> abstol) .& (abs.(testval) .> reltol .* abs.(integrator.u))) && (return false)
     end
-    return true
+
+    if min_t === nothing
+        return true
+    else
+        return integrator.t >= min_t
+    end
 end
 
 # Allow user-defined tolerances and test functions but use sensible defaults
 # test function must take integrator, time, followed by absolute
 #   and relative tolerance and return true/false
-function TerminateSteadyState(abstol = 1e-8, reltol = 1e-6, test = allDerivPass)
-    condition = (u, t, integrator) -> test(integrator, abstol, reltol)
+function TerminateSteadyState(abstol = 1e-8, reltol = 1e-6, test = allDerivPass;
+                              min_t = nothing)
+    condition = (u, t, integrator) -> test(integrator, abstol, reltol, min_t)
```

Comment:
Is this breaking? The number of arguments for the user supplied `test` changed. This is causing test failures for me. The release only bumped from 2.19 to 2.10.

Reply:
I didn't know that was an exposed interface.

Reply:
```julia
    ss_test(int, atol, rtol) = ###
    cb = TerminateSteadyState(1e-3, 1e-4, ss_test)
    sol = solve(prob, Tsit5(), callback = cb)
```
now requires a 4 arg test function